### PR TITLE
refactor(context): PantherContext improved

### LINKED ISSUE #11: Problem with "wait" context
version: 0.4.0

```gherkin
Then I wait for "#element" to be visible
```

calls `PantherContext::iWaitForElementToBeVisible()`

which calls `$driver->wait(3000, WebDriverExpectedCondition::visibilityOfAnyElementLocated(WebDriverBy::cssSelector($element)));`

but `PantherDriver::wait()` must be a string...

### PR DESCRIPTION
**Pull request by Guikingone**

### CODE REVIEW

File: src/Context/PantherContext.php [hunk 1]
```diff
@@ -94,7 +92,7 @@ public function iWaitForElementToBeVisible(string $element): void
         $driver = $this->getDriver();
 
         try {
-            $driver->wait(3000, WebDriverExpectedCondition::visibilityOfAnyElementLocated(WebDriverBy::cssSelector($element)));
+            $driver->getWaitHelper()->waitFor($element);
```
Comment (nikophil) [inline src/Context/PantherContext.php]: why don't you create `WaitHelper::waitForVisibility()` (as well as `waitForInvisibility()`)?

File: tests/Driver/PantherDriverTest.php [hunk 2]
```diff
@@ -6,6 +6,8 @@
 
 use Behat\Mink\Exception\UnsupportedDriverActionException;
 use Facebook\WebDriver\WebDriver;
+use Facebook\WebDriver\WebDriverBy;
+use Facebook\WebDriver\WebDriverExpectedCondition;
```
Comment (nikophil) [inline tests/Driver/PantherDriverTest.php]: this seems to not be useful

**Review by nikophil — CHANGES_REQUESTED** [review]

File: tests/Driver/PantherDriverTest.php (hunk 2 above)
Comment (Guikingone) [reply tests/Driver/PantherDriverTest.php]: Fixed 🙂

File: src/Context/PantherContext.php (hunk 1 above)
Comment (Guikingone) [reply src/Context/PantherContext.php]: Created 🙂

**Review by nikophil — APPROVED** [review]

**Pull request merged**